corehq/apps/export/templates/export/partials/table.html

```diff
@@ -353,7 +353,7 @@ <h4 class="modal-title">
                data-bind="if: emailedExport.taskStatus && emailedExport.taskStatus.failed() == 'too_large'">
               <i class="fa-solid fa-triangle-exclamation"></i>
               {% blocktrans with max_rows=max_daily_export_size|intcomma %}
-                Your export exceeds the limit of {{ max_rows }} rows. Please see <a href="https://confluence.dimagi.com/display/commcarepublic/Selecting+Tools+for+CommCare+Data+Processing+and+Analysis" data-toggle="modal">documentation</a> for more information.
+                Your export exceeds the limit of {{ max_rows }} rows. Please see <a href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143957751/Data+Analysis+Visualization" data-toggle="modal">documentation</a> for more information.
```

Comment:
I think https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143956271/Form+Data+Export#Multimedia-Exports.1 is the correct link here? It specifically references the file size limitation

Reply:
@dreesefadil @mjriley the correct link here is https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2337669143/Setting+up+a+Data+Pipeline+via+CommCare

That error message pops up when the amount of data in a UI based export has exceeded 100,000 rows. The one that Matt pointed to is about the size of the multimedia export, which is a separate error message.